src/js/plugins/plugin.resume.js

```diff
@@ -44,10 +44,11 @@ BookReader.prototype.getResumeValue = function() {
  * Can be overriden for different implementation
  *
  * @param {Number} index leaf index
- * @param {String} cookieName
+ * @param {string} [cookieName]
  */
-BookReader.prototype.updateResumeValue = function(index, cookieName='br-resume') {
+BookReader.prototype.updateResumeValue = function(index, cookieName) {
   const ttl = new Date(+new Date + 12096e5); // 2 weeks
-  const path = this.options.rsumeCookiePath || window.location.pathname;
+  const path = this.options.resumeCookiePath || window.location.pathname;
+  cookieName = cookieName || 'br-resume';
```

Comment:
I'd suggest assigning to a new local variable, as our usual ESLint rules prevent parameter reassignment https://eslint.org/docs/rules/no-param-reassign

Reply:
:+1: fixed